cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -103,6 +103,14 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
 		return err
 	}
 
+	// If this is the last etcd member of the cluster, it cannot be removed due to "not
+	// enough started members". Skip it as the cluster will be destroyed in the next phase.
+	etcdClientAddress := etcdutil.GetClientURL(&cfg.LocalAPIEndpoint)
+	if len(etcdClient.Endpoints) == 1 && etcdClient.Endpoints[0] == etcdClientAddress {
+		klog.V(1).Info("[etcd] This is the last etcd member of etcd cluster, skip removing it")
```

Comment:
please change to:
`klog.V(1).Info("[etcd] This is the only remaining etcd member in the etcd cluster, skip removing it")`

Reply:
done, thanks.